app/frontend/components/domains/requirements-library/requirements-block-modal/index.tsx

```diff
@@ -105,7 +111,7 @@ export const RequirementsBlockModal = observer(function RequirementsBlockModal({
         }
       }
 
-      return returnValue
+      return getPrunedOptionsMapperComplianceConfiguration(returnValue, autoComplianceModuleConfigurations)
```

Comment:
rename returnValue if your're not returning it?

Reply:
good catch